e2e_playwright/shared/app_utils.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Streamlit Inc. (2018-2022) Snowflake Inc. (2022-2024)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import annotations
+
+from playwright.sync_api import Locator, expect
+
+
+def expect_prefixed_markdown(
```

Comment:
thanks for extracting the utils functions. I hoped that it could also be used for the `expect_written_text` function in the `st_altair_chart_basic_select` function [here](https://github.com/streamlit/streamlit/blob/develop/e2e_playwright/st_altair_chart_basic_select_test.py#L56), so I believe it would make sense to offer it directly in a way where it'd be compatible. But can also be done in a future PR

Reply:
Updated st_altair_chart_basic_select to use the utils method 👍